pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/json/MutableJsonIndexImpl.java

```diff
@@ -350,58 +354,48 @@ private RoaringBitmap getMatchingFlattenedDocIds(Predicate predicate) {
     }
   }
 
-  @Override
-  public Map<String, RoaringBitmap> getMatchingDocsMap(String key) {
-    Map<String, RoaringBitmap> matchingDocsMap = new HashMap<>();
+  @VisibleForTesting
+  public Map<String, RoaringBitmap> convertFlattenedDocIdsToDocIds(Map<String, RoaringBitmap> valueToFlattenedDocIds) {
```

Comment:
remove public?

Reply:
This is now used to optimize SV extraction path by converting flattened doc IDs map to unflattened.